INN/umbrella-ipbs
# Hotfixes for the homepage layout in response to bigger texts

## Changes

This pull request makes the following changes:

- increases the breakpoints where:
    - the homepage header now goes to reduced-padding at 950px instead of 900px
    - the homepage header now goes to mobile at 829px instead of 770px

### Before:

![Screen Shot 2020-06-24 at 16 06 13](https://user-images.githubusercontent.com/1754187/85625540-a40e1980-b639-11ea-9961-67800f05ba13.png)

### After
950px:
![Screen Shot 2020-06-24 at 16 33 56](https://user-images.githubusercontent.com/1754187/85625586-b12b0880-b639-11ea-907d-aee301bbfb45.png)
949px:
![Screen Shot 2020-06-24 at 16 34 03](https://user-images.githubusercontent.com/1754187/85625592-b38d6280-b639-11ea-9f7e-22db3b8e507b.png)
830px
![Screen Shot 2020-06-24 at 16 34 13](https://user-images.githubusercontent.com/1754187/85625612-ba1bda00-b639-11ea-8ef0-6174c64f8a3f.png)
829px:
![Screen Shot 2020-06-24 at 16 34 28](https://user-images.githubusercontent.com/1754187/85625617-bc7e3400-b639-11ea-828a-31b258525022.png)



## Why



Because the text currently in use on ipbs.staging.wpengine.com uses 

## Testing/Questions

Features that this PR affects:

- the homepage


Questions that need to be answered before merging:

- [x] Does this pass the tests in point 3 below?

Steps to test this PR:

1. Copy the text from http://ipbs.staging.wpengine.com/ into the text widget in the "Homepage Top" widget area
2. Copy the menu item labels from http://ipbs.staging.wpengine.com/ into the menu items in the "Homepage Quick Links" menu
3. View the homepage, slowly resizing it from >1200px to <700px. 
    - the blue box should never cover the main nav
    - the blue box should not get too close to the borders of the image.
